src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -98,20 +98,20 @@ RCLConsensus::Adaptor::Adaptor(
     JLOG(j_.info()) << "Consensus engine started (cookie: " +
             std::to_string(valCookie_) + ")";
 
-    if (validatorKeys_.nodeID != beast::zero)
+    if (validatorKeys_.nodeID != beast::zero && validatorKeys_.keys)
```

Comment:
@scottschurr  Hello Scott, thank you for working on the commit yourself, that was very sweet of you.

I noticed that you didn't include the first condition in your work, I missed that :) thanks

Reply:
I'll update it along with the forthcoming PR comments, noting it here ^